docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Quickstart
+hide_title: false
+sidebar_position: 2
+---
+
+**Ready to give Apache Superset a try?** This quickstart will help you run Superset on your local machine in **5 simple steps**.
+It assumes that you have [Docker](https://www.docker.com) installed.
+
+### 1. Get Superset
+To get started, add the latest Superset version in the `SUPERSET_VERSION` environment variable, and pull the container from Docker Hub:
+```
+$ export SUPERSET_VERSION="";
+         docker pull apache/superset:$SUPERSET_VERSION
+```
+:::tip
+Not sure what the latest stable Superset version is? [Click here](https://github.com/apache/superset/releases) to check our release page 
+for up-to-date information.
+:::
```

Comment:
Maybe this would be more intuitive?

To get started, set the `SUPERSET_VERSION` environment variable with the latest Superset version. [Click here](https://github.com/apache/superset/releases) to check the latest version.
```
$ export SUPERSET_VERSION=<latest_version>
```

Pull the Superset image from Docker Hub:
```
$ docker pull apache/superset:$SUPERSET_VERSION
```

Reply:
Yeah that's better, I just changed the first step to it
<img width="863" alt="image" src="https://github.com/apache/superset/assets/45772806/ebfbbab9-c884-4382-94a6-8b304c151355">